code/obj/critter/pets_small_animals.dm

```diff
@@ -948,16 +961,21 @@ var/list/shiba_names = list("Maru", "Coco", "Foxtrot", "Nectarine", "Moose", "Pe
 		src.learned_words += learning_word
 		DEBUG_MESSAGE("[src]: chosen word: [learning_word]")
 
-	proc/chatter()
+	proc/chatter(var/sing=0)
 		if (islist(src.learned_phrases) && src.learned_phrases.len && prob(20))
 			src.say(pick(src.learned_phrases))
 		else if (islist(src.learned_words) && src.learned_words.len)
 			var/my_word = pick(src.learned_words) // :monocle:
-			src.say("[capitalize(my_word)][pick(".", "!", "?", "...")]")
+			my_word = "[capitalize(my_word)][pick(".", "!", "?", "...")]"
+			var/quote = "\""
+			if (sing)
+				quote = "<img class=\"icon misc\" style=\"position: relative; bottom: -3px; \" src=\"[resource("images/radio_icons/note.png")]\">"
```

Comment:
seems like this would be better done with a prepend of the var quote instead of an override?

Reply:
not sure exactly what you mean